e2e/testcases/helm_sync_test.go

```diff
@@ -64,6 +64,18 @@ func TestPublicHelm(t *testing.T) {
 	if err := nt.Validate("my-ingress-nginx-controller", "ingress-nginx", &appsv1.Deployment{}); err != nil {
 		nt.T.Error(err)
 	}
+	// reset rs to sync from original git repo
+	nt.MustMergePatch(rs, fmt.Sprintf(`{"spec": {"sourceType": "%s", "helm": null, "git": {"dir": "acme", "branch": "main", "repo": "%s", "auth": "ssh","gcpServiceAccountEmail": "", "secretRef": {"name": "git-creds"}}}}`,
+		v1beta1.GitSource, origRepoURL))
+	nt.WaitForRepoSyncs()
+	nt.T.Log("Update RootSync to sync from a public Helm Chart without specified release namespace")
+	nt.MustMergePatch(rs, fmt.Sprintf(`{"spec": {"sourceType": "%s", "helm": {"repo": "%s", "chart": "ingress-nginx", "auth": "none", "version": "4.0.5", "releaseName": "my-ingress-nginx"}, "git": null}}`,
+		v1beta1.HelmSource, publicHelmRepo))
+	nt.WaitForRepoSyncs(nomostest.WithRootSha1Func(helmChartVersion("ingress-nginx:4.0.5")),
+		nomostest.WithSyncDirectoryMap(map[types.NamespacedName]string{nomostest.DefaultRootRepoNamespacedName: "ingress-nginx"}))
+	if err := nt.Validate("my-ingress-nginx-controller", configsync.DefaultHelmReleaseNamespace, &appsv1.Deployment{}); err != nil {
```

Comment:
Suggest validating `my-ingress-nginx-controller` doesn't exist in the ingress-nginx namespace anymore.

Reply:
Done.